src/Domain/Entities/ItemType.cs

```diff
@@ -0,0 +1,12 @@
+namespace Domain.Entities;
+
+public class ItemType
+{
+    ItemTypeId Id { get; set; } 
+    string Name { get; set; }
+    string? Description { get; set; } 
+    float QualityMin { get; set; } // the minimum quality this item appears in
+    float QualityMax { get; set; } // the maximum quality this item can have
+    bool StatTrackAvailable { get; set; }// whether or not stat track is available
+    string ImageUrl { get; set; }
+}
```

Comment:
add the appropriate methods, according to: [the methods section](https://github.com/ddjerqq/Astro/issues/6#methods)